src/lib/operators/abstract_aggregate_operator.cpp

```diff
@@ -29,24 +29,24 @@ const std::vector<std::shared_ptr<AggregateExpression>>& AbstractAggregateOperat
 const std::vector<ColumnID>& AbstractAggregateOperator::groupby_column_ids() const { return _groupby_column_ids; }
 
 std::string AbstractAggregateOperator::description(DescriptionMode description_mode) const {
-  const auto* const separator = description_mode == DescriptionMode::SingleLine ? " " : "\n";
+  const char separator = (description_mode == DescriptionMode::SingleLine ? ' ' : '\n');
```

Comment:
Is the `char` necessary?

Reply:
No, but it may be more space-efficient?

Reply:
I think, the question is not about `char` vs `char*` but more whether `const auto separator` would work.

Reply:
Got it, thanks! 😄